ref/src/yppm_core.F90

```diff
@@ -21,11 +21,10 @@
 
 module yppm_core_mod
 
-  implicit none
-
-  private
```

Comment:
It is usually best practice to make module variables and routines private by default and then use `public` to expose the ones needed outside of the module.  I do not understand your comment about these being unnecessary.  Can you please explain?

Reply:
@christopherwharrop-noaa I removed the ‘private’ from this module in following the example of sw_core.F90. Now that all of the variables and routines in this module need to be exposed to yppm_driver, would we need to declare all of them as `public` and  `use` all of them by name in yppm_driver module? That would add quite a bit of redundancy, but if that is what you’d suggest, I will certainly remove this change.  I did not give this removal too much consideration, and do not have experience with the best practices/pros & cons of keeping specific variables/routines private and public. If it is best practice, and you would suggest keeping the module variables/routines private, I can leave the `private` statement and make futher adjustments to accommodate for this behavior (if any) as needed.

Reply:
@robgonzalezpita - Thank you for explaining.  You are correct that the number of things that the driver needs would result in a long `only` clause.  However, some of the routines in this module are for internal use only, and should be private.  I would recommend we keep the routines `private` by default and use `public` to expose the ones needed outside.  And then, for now, `use` the module without the normally desired `only` clause to keep that simple.